cmd/sebak/cmd/wallet/payment.go

```diff
@@ -153,12 +164,13 @@ func init() {
 ///   kpDest   = Newly created account's address
 ///   amount   = Amount to send as initial value
 ///   seqid    = SequenceID of the last transaction
+///   target   = Address of the linked account, if we're creating a frozen account
 ///
 /// Returns:
 ///   `sebak.Transaction` = The generated `Transaction` creating the account
 ///
-func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64) transaction.Transaction {
-	opb := transaction.NewOperationBodyCreateAccount(kpDest.Address(), amount)
+func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64, target string) transaction.Transaction {
+	opb := transaction.NewOperationBodyCreateAccount(kpDest.Address(), amount, target)
```

Comment:
Does target means general account linked to frozen account?

Reply:
Yes